pyproject.toml

```diff
@@ -168,6 +168,14 @@ filterwarnings = [
     "ignore:path is deprecated\\. Use files\\(\\) instead:DeprecationWarning",
     "ignore:This process \\(.+\\) is multi-threaded",
 ]
+markers = [
+    "slow: mark Falcon tests as slower (potentially taking more than ~500ms).",
+]
 testpaths = [
     "tests"
 ]
+
+[tool.cibuildwheel]
+build-frontend = "build"
+test-requires = "-rrequirements/cibwtest"
```

Comment:
Is a space miss here? Between -r and yhr requirement file

Reply:
No miss, this is an ugly workaround by design. I did have a space in the first iteration, but cibw was detecting it and splitting as two requirements despite double quoting. It works via an envvar but not via TOML. If you know how to add that space, happy to change.

OTOH Pip takes it fine without space too, so maybe it is not a big deal? I can add a `NOTE`.

Reply:
A note sounds good. Maybe there is an option to pass a requirement file instead?

Reply:
I haven't found any, I just found some example repositories passing with or without space via `CIBW_*` envvars.

Now that I have narrowed down that list, we could also list them inside a TOML list, but I think it is cleaner to keep it in a req file maintenance wise?

Reply:
Makes sense to have a file. Probably the same wrt numpy. Let's just add a bote saying "do not add the space here since ..."

Reply:
I'll try one more TOML approach, and will add a `NOTE` if even that fails.
Moreover, it is not strange to use the `-rrequirements.file` form, it is in fact quite usual that arguments following a single-letter shorthand option follow without a space, and ones following `--full-option-name` do have a space inbetween.

Reply:
OK this worked actually
```toml
test-requires = ["-r requirements/cibwtest"]
```